### ISSUE TITLE
DJI account phone verification doesn't work on iPads with large screens

### ISSUE THREAD

**Issue by byuarus** [dji-sdk/Mobile-SDK-iOS#436]
Hello,
I'm working on integrating NFZ unlock workflow in my app. In order to unlock authorization NFZ I have to call 
```
DJISDKManager.userAccountManager().logIntoDJIUserAccount(withAuthorizationRequired: true, withCompletion: nil)
```
It looks that depending on the iOS device that I use, DJI SDK will open different a desktop or a mobile version of the login page. 
For all iPhones and iPad mini I see the mobile version of login page and the whole workflow for account verification works as expected.

When I use any of the big iPads (iPad Air(8th gen), iPad Pro 9.7 incn, iPad Pro 11 inch or iPad Pro 12.9 inch) DJI will present a different login web page (I assume it's a desktop version). After I provide my credentials on the next screen, there is an endless spinner where I should select a country phone code. After I type in my phone number and when I click "Request" button nothing happens. Because of the issue users are blocked and cannot verify their DJI accounts using DJI SDK on large iPads.

![IMG_0160](https://user-images.githubusercontent.com/10615444/105909007-d458d480-5fdb-11eb-82c8-c54163c20121.PNG)
![IMG_22F28A0CF97C-1](https://user-images.githubusercontent.com/10615444/105909051-e3d81d80-5fdb-11eb-842f-de8172edb9e4.jpeg)

Environment:
- DJI SDK: 4.13.1, 4.14-trail 1, 4.14-trail 2
- XCode: 12.2
- iPad Air(8th gen), iPad Pro 9.7 incn, iPad Pro 11 inch or iPad Pro 12.9 inch (repo steps work on XCode simulator and on real devices)
- iOS 13.6, iOS 14.2

Repo steps:
- create a new DJI account or use unauthorized DJI account
- call `DJISDKManager.userAccountManager().logIntoDJIUserAccount(withAuthorizationRequired: true, withCompletion: nil)`

**Comment by dji-dev** [comment]
Agent comment from Luce Luo in <a href='https://djisdksupport.zendesk.com/agent/tickets/43403'>Zendesk ticket #43403</a>:<br><br>Dear Customer,

Thank you for contacting DJI.
Sorry, I did reproduce this issue and have escalated this to the engineer.
It will cost some time to investigate and will contact you again once any progress.

Thanks,

Luce Luo
DJI Developer Support

**Comment by JoshSearle** [comment]
Hello, 

We are seeing this issue also.  Has there been any progress made in getting a fix out?

**Comment by JoshSearle** [comment]
Hello,

We are seeing this issue also. Has there been any progress made or is there a timeframe as to when this might be fixed?

**Comment by JoshSearle** [comment]
Hello,

We are seeing this issue also. Has there been any progress made or is there a timeframe as to when this might be fixed?

**Comment by JoshSearle** [comment]
Hello,

We are seeing this issue also. Has there been any progress made or is there a timeframe as to when this might be fixed?

**Comment by dji-dev** [comment]
Agent comment from Luce Luo in <a href='https://djisdksupport.zendesk.com/agent/tickets/43403'>Zendesk ticket #43403</a>:<br><br>Dear Customer,

Thank you for contacting DJI.
The engineer still working on this issue but has no more progress for the being.
We can't guarantee the exact date but will update here if any progress.

Thanks,

Luce Luo
DJI Developer Support<br><p><em><small>°°°</small></em></p>

**Comment by dji-dev** [comment]
Agent comment from Luce Luo in <a href='https://djisdksupport.zendesk.com/agent/tickets/43403'>Zendesk ticket #43403</a>:<br><br>Dear Customer,

Thank you for contacting DJI.
This problem has been fixed in the 4.15 version and has been released.

Thanks,

Luce Luo
DJI Developer Support
<br><p><em><small>°°°</small></em></p>
